src/main.controller.js

```diff
@@ -967,6 +1040,16 @@ plugin.controller('wgnConfigCtrl', ['$scope', '$q', '$routeParams', 'znData', 'z
 			});
 		}
 
+		function doSaveSecure (secure) {
+
+			var configId = $scope.settings.multi && $scope.editing.config.$id;
```

Comment:
You actually can't use it in single mode because of the way the firebase settings are stored. I have an issue in for normalizing single and multi to use the same structure for this and a couple of other reasons.

It will throw an error if you try to configure single mode + secure, so it can't get this far.

Reply:
that's really unfortunate. Probably not worth refactoring at this point to make single/multi more inline with each other though.